e2e/stress/IoTClientPerf/Scenarios/GenerateIotHubConfigTest.cs

```diff
@@ -0,0 +1,87 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.Devices.E2ETests
+{
+    public class GenerateIotHubConfigTest : PerfScenario
+    {
+        // Pattern: <NamePrefix>_<auth>_id
+        // - SAS key devices generated will have the same key as the device specified in the IOTHUB_DEVICE_CONN_STRING/IOTHUB_DEVICE_CONN_STRING2.
+        // - X509 key devices generated will have the same certificate as specified in IOTHUB_X509_PFX_CERTIFICATE
+        private static StreamWriter s_outputFile = new StreamWriter("devices.txt");
+        private static SemaphoreSlim s_semaphore = new SemaphoreSlim(1);
+
+        static GenerateIotHubConfigTest()
+        {
+            Console.WriteLine("----------------------");
+            Console.WriteLine("To import the generated file:");
+            Console.WriteLine("1. Create an Azure Storage Container");
+            Console.WriteLine("2. Get a container SAS key:");
```

Comment:
Reverse the order of step 2 and 3 - so that we generate the container SAS key, and then use it immediately after.